schema.sql

```diff
@@ -0,0 +1,8 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (id SERIAL, 
```

Comment:
- [OPTIONAL] Please make sure the `id` can not be empty and it should be unique.